docs/troubleshooting/create-a-minimal-reproduction.mdx

```diff
@@ -0,0 +1,104 @@
+- [Rails](https://github.com/clerkinc/clerk-rails-starter)
+- [Chrome Extension](https://github.com/clerkinc/clerk-chrome-extension-starter)
+
+## Steps to create a minimal reproduction
+
+<Steps>
+
+### Clone a template
+
+Clone down one of the [templates listed above](#templates) in order to have the application locally on your machine.
+
+In the example below, you can see options for cloning the [Clerk Next.js App Router](https://github.com/clerk/clerk-nextjs-app-quickstart) repository.
+
+<Images 
+  width={3024}
+  height={1654}
+  src="/docs/images/troubleshooting/how-to-clone.webp"
+  alt="The 'clerk-nextjs-app-quickstart' repository page with the 'Code' button clicked on and its drop down menu is opened. There is a red arrow pointing to the 'Code' button."
+/>
+
+### Create a Clerk application
+
+Create a new Clerk application through the Clerk Dashboard. You can follow Clerk's [setup guide](https://clerk.com/docs/quickstarts/setup-clerk) to help you get started. 
+
+### Set the environment keys
+
+Set the environment keys in your Clerk application. You can find instructions on how to do so in the appropriate [quickstart guide](/docs/quickstarts/overview).
+
+### Run the app
+
+Run the application locally and verify that it’s functional. For example, if you used `npm` to install the dependencies, you can run the app's development instance with the command `npm run dev`. The app should then be running on `http://localhost:3000` with no errors.
+
+### Add code to reproduce the issue
+
+Adjust your template project by adding code gradually until you get to the point where you are running into the same error.
+
+### Add a README file (optional)
+
+Add a README file to the root of the repository that contains the steps that must be taken in the app in order to produce the error (click X button, log in using Y provider, etc).
+
+<Callout type="warning">
+  This step is only required if you are submitting the minimal reproduction to the Clerk support team. 
+</Callout>
+
+</Steps>
+
+It’s important throughout this process to be vigilant about removing code that is not necessary to produce the issue. The ideal result is that the repository gets to the state where the exact same issue is able to be produced, with the fewest lines of code possible to get to that point (beyond the baseline included in the starter templates).
+
+Often times, the process of creating the minimal reproduction will clarify or resolve the original issue. This is great, if it does happen! That is the magic of minimal reproductions.
+
+## Submitting a minimal reproduction
+
+The Clerk support team **requires** minimal reproductions in order to start looking into reported bugs or issues for any users or customers that are not on an enterprise plan. 
+
+### Submission requirements
+
+Before submitting a minimal reproduction to the Clerk support team, ensure that the repository:
+- Is based off one of the starter templates above
```

Comment:
Suggestion: "Is based off one of the starter templates above or a very minimal recreation of the issue"

Reply:
Its not unreasonable for a customer to strip down their repo to a minimal reproduction. I think advanced customers will look over the steps and understand the goal and attempt to achieve it. We have definitely looked at reproductions that were not built from the ground up.

Reply:
Fair! Copy is updated :)